frontend/src/store/calendar/reducer.ts

```diff
@@ -0,0 +1,26 @@
+import { Reducer } from 'redux'
+import { ActionTypes, CALENDAR_ACTIONS } from './types'
+
+const initialState = {
+  isClicked: false,
+  clickedDate: 0,
+}
+
+type State = {
+  isClicked: boolean
+  clickedDate: number
+}
+
+export const calendar: Reducer<State, ActionTypes> = (state = initialState, action) => {
+  switch (action.type) {
+    case CALENDAR_ACTIONS.SET_IS_CLICKED: {
+      return {
+        ...state,
+        isClicked: action.newIsClicked,
```

Comment:
try using more descriptive for state names!
`isClicked` may mean `hasSelectedDate`, it may mean user deleted something, selected something, but no idea what. 

clicked Date on the other hand is clear :D

Reply:
Thanks Junxiang! Good learning point for me 😄